### DISCUSSION TITLE
CORS issue

### DISCUSSION THREAD

**Discussion by Happy-Ferret** [facebook/docusaurus#6868] [Q&A]
I'm working on a web app that deploys a smart contract.
The frontend is supposed to be part of a Docusaurus page.

Unfortunately, I'm running into a CORS issue while attempting to load the truffle-contract.min.js.

All the scripts are loaded as part of the `scripts` array inside `docusaurus.config.js`.

Any idea on how to overcome this?

```
truffle-contract.min.js:1 Access to XMLHttpRequest at 'webpack-internal:///./node_modules/react/cjs/react.development.js' from origin 'http://localhost:3000' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, chrome, chrome-extension, chrome-untrusted, https.
(anonymous) @ truffle-contract.min.js:1
truffle-contract.min.js:1          Failed to load resource: net::ERR_FAILED
```

**Comment by slorber** [comment]
It's impossible to help if you don't provide source code + deployment to inspect 😅
